pallets/author-slot-filter/src/migration.rs

```diff
@@ -0,0 +1,50 @@
+use core::marker::PhantomData;
+use frame_support::storage::migration;
+use frame_support::traits::Get;
+use frame_support::traits::OnRuntimeUpgrade;
+use frame_support::weights::Weight;
+use sp_runtime::Percent;
+
+use super::num::NonZeroU32;
+use super::pallet::Config;
+
+pub struct EligibleRatioToEligiblityCount<T>(PhantomData<T>);
+
+const PALLET_NAME: &[u8] = b"AuthorSlotFilter";
```

Comment:
I would put these inside the migration. Or at least change the naming. In the future OLD_ITEM_NAME might refer to multiple things

Reply:
aren't these already in the migrations module? \
Alternatively maybe we could use `ELIGIBLE_RATIO_ITEM_NAME` & `ELIGIBLE_COUNT_ITEM_NAME`?